docs/en/operations/settings/settings.md

```diff
@@ -981,7 +981,7 @@ Default value: 0.
 - Type: seconds
 - Default value: 60 seconds
 
-Controls how fast errors of distributed tables are zeroed. Given that currently a replica was unavailabe for some time and accumulated 5 errors and distributed_replica_error_half_life is set to 1 second, then said replica is considered back to normal in 3 seconds since last error.
+Controls how fast errors in distributed tables are zeroed. If a replica is unavailabe for some time, accumulates 5 errors, and distributed_replica_error_half_life is set to 1 second, then the replica is considered normal 3 seconds after last error.
```

Comment:
Don't understand the meaning of this change.

Reply:
Just edits from the native speaker.